### ISSUE TITLE
wiki - Setup AP_SYSID

### ISSUE THREAD

**Issue by Clooney82** [athertop/MavLink_FrSkySPort#14]
Hi Paul,

while setting up swarming on Missionplanner I changed the parameter **SYSID_THISMAV**, to identify the different copters.
This led to a none working teensy telemetry.
It took me about 2 weeks to figure out why this happend, I almost though my teensy was faulty.

But afterwards the solution was too simple to see it immediatly.
I only needed to change line 121 in MavLink_FrSkySPort.ino
`#define AP_SYSID            1       // autopilot system id
`
to the value of **SYSID_THISMAV**.

Maybe this Information should be added to the wiki on setup page.
If telemetry not working anyone should check if **AP_SYSID** and **SYSID_THISMAV** matches.

BR, Jochen

**Comment by athertop** [comment]
Jochen, so how does line 121 now read? And will the change work for all users (even if not involved with swarm setup)?

**Comment by Clooney82** [comment]
Hi Paul,

it is not a code change needed. Only as hint for the wiki page.
Default should work out of the box.
It is only in case some users change **SYSID_THISMAV** in ArduCopter/Plane, then they must also change **AP_SYSID** to the same value, otherwise telemetry will not work.
This is because Teensy code will only look for MAVLink message with SYSID 1 (which is default value of SYSID_THISMAV in ArduCopter/Plane)

e.g. I changed APM Parameter SYSID_THISMAV to 59, so I had to change AP_SYSID in teensy code also to 59 to get telemetry working again.

BR, Jochen
